BlogTemplate/Models/BlogDataStore.cs

```diff
@@ -73,7 +75,9 @@ public void IterateComments(IEnumerable<XElement> comments, List<Comment> listAl
                     AuthorName = comment.Element("AuthorName").Value,
                     Body = comment.Element("CommentBody").Value,
                     AuthorEmail = comment.Element("AuthorEmail").Value,
-                    PubDate = DateTime.Parse((comment.Element("PubDate").Value), culture, System.Globalization.DateTimeStyles.AssumeLocal)
+                    PubDate = DateTime.Parse((comment.Element("PubDate").Value), culture, System.Globalization.DateTimeStyles.AssumeLocal),
+                    IsPublic = Convert.ToBoolean(comment.Element("IsPublic").Value),
+                    UniqueId = (Guid.Parse(comment.Element("UniqueId").Value))
```

Comment:
nit: put a comma at the end, so that this line doesn't end up in a diff if you add more properties later (like line 78 above)